tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivativeImpl.tpp

```diff
@@ -908,7 +916,15 @@ void test_impl(const Spectral::Quadrature quadrature,
   ::InverseJacobian<DataVector, Dim, Frame::Logical, Frame::Inertial> inv_jac{
       mesh.number_of_grid_points(), 0.0};
   for (size_t i = 0; i < Dim; ++i) {
-    inv_jac.get(i, i) = 1.0;
+    inv_jac.get(i, i) = 2.0;
+  }
+  const auto det_inv_jacobian = determinant(inv_jac);
+  const auto jacobian = determinant_and_inverse(inv_jac).second;
+
+  const auto logical_coords = logical_coordinates(mesh);
+  tnsr::I<DataVector, Dim, Frame::Inertial> inertial_coords{};
+  for (size_t i = 0; i < logical_coords.size(); ++i) {
+    inertial_coords[i] = logical_coords[i];
```

Comment:
If inertial == logical, why is inv_jac = 2 just above?

Reply:
basically a way to set up a configuration that creates rather unique output. non-identity Jacobian allows testing Jacobian terms, identity map makes the by-hand math easier :shrug: I'll add a comment explaining that :P